Livida Blog/dist/metadata.json

```diff
@@ -0,0 +1,21 @@
+{
+    "author": {
+      "name": "Livida",
+      "id": "559160676485169161"
```

Comment:
is this your discord name and id?

Reply:
It's the account for my community but I can change it to my own if needed.

Reply:
usually its your own account, since it would be used if the presence is broken for contacting the dev.

Reply:
Ah alright, I'll change to my own then.